core/hydromet.py

```diff
@@ -136,7 +136,7 @@ def build_precip_table(geo_df: geoDF, all_zips_list: list, noaa_url: str,
         with ZipFile(memfile, 'r') as openzip:
             gridfiles = openzip.namelist()
             mes = "Expected to find 1 file, found {0}".format(len(gridfiles))
-            assert len(gridfiles) == 1, mes
+            assert len(gridfiles) == 3, mes
```

Comment:
Is there a reason you changed this to 3? You are always selecting the first gridfile in the next line. If there are three files, we should probably determine why and make sure we are selecting the correct one.